x/protocolpool/keeper/keeper.go

```diff
@@ -30,6 +30,7 @@ type Keeper struct {
 	// State
 	Schema         collections.Schema
 	BudgetProposal collections.Map[sdk.AccAddress, types.Budget]
+	ContinuousFund collections.Map[sdk.AccAddress, types.ContinuousFund]
```

Comment:
Can we add the same docs that continuous funds can be one at time per address in the module README?